src/components/About/About.css

```
@@ -4,4 +4,57 @@
   align-content: center;
   align-items: center;
   min-height: 300px;
+  flex-direction:column;
 }
+
+.about_bg{
+  background-image: url('./images/ss_2.jpg');
+  width: 100%;
+  height: 400px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: right;
+}
+
+.about__bg-overlay{
+  width: 100%;
+  height: 400px;
+  background: rgba(40, 167, 69, 0.6);
+  color: #fff;
+  text-align: center;
+  padding-top: 2rem;
```

```suggestion
  padding-top: 2rem;
  border-radius: inherit;
```
Follow up from the above message.